docs/source/usage/build-public.rst

```diff
@@ -16,6 +16,11 @@ build-public
     The actions are performed in this order, meaning that *internal* properties will be taken into account when matching the JSON schema.
     If a component containing nested components is deleted, those nested components are deleted as well.
 
+    .. warning::
```

Comment:
Please also add "However, this behavior does not affect the deletion of any property, which starts with the name ``internal:``", as otherwise it could be misleading.

Further, please also change the order of the bullet points at the beginning of the file, as the deletion always happen, wheras the schema is optional:
![image](https://github.com/user-attachments/assets/d069d7a1-f299-4efa-9511-e1e372ebcb95)